package-lock.json

```diff
@@ -1,1279 +1,2297 @@
 {
   "name": "@oat-sa/eslint-config-tao",
   "version": "1.2.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 3,
```

Comment:
**issue:** Please revert also the lock file. For the sake of the update, it needs to be generated using `npm 6`, until we moved all the packages to `npm 7+` otherwise it may break the install using `npm ci`

Reply:
reverted

Reply:
I still see lockfileVersion 3, and installing with npm 6 causes it to be reverted to 1. I think 1 should be kept.

Reply:
my bad, reverted to 1